pkg/archive/deb.go

```diff
@@ -63,23 +62,23 @@ var (
 	}
 )
 
-type debArchive struct {
+type DebPackager struct {
```

Comment:
Curious, why make these public? The fields are private and they're only ever used to call `Package(...)`, which is the public interface. Is it a style/correctness thing or is there another reason?